packages/fuels-programs/src/contract.rs

```diff
@@ -175,17 +167,46 @@ impl DeployConfiguration {
 /// compiling, deploying, and running transactions against a contract.
 /// The contract has a wallet attribute, used to pay for transactions and sign them.
 /// It allows doing calls without passing a wallet/signer each time.
```

Comment:
```suggestion
/// It allows doing calls without passing a wallet/signer each time.
#[derive(Debug, Clone, Default)]
```

Reply:
`Contract` doesn't have a wallet attribute

Reply:
Good catch! That is a leftover from the account PR.